src/styles/utils.scss/normalize.scss

```diff
@@ -0,0 +1,12 @@
+.page {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font__size;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
```

Comment:
These styles cannot be written in `normalize.scss`
Move them to separate block styles file.
And remember, that according to BEM methodology, we cannot position blocks, only elements.